pkg/cvo/sync_worker.go

```diff
@@ -200,12 +200,149 @@ func (w *SyncWorker) StatusCh() <-chan SyncWorkerStatus {
 	return w.report
 }
 
+func (w *SyncWorker) syncPayload(ctx context.Context, work *SyncWork, reporter StatusReporter, clusterVersion *configv1.ClusterVersion) error {
+	desired := configv1.Release{
+		Version: work.Desired.Version,
+		Image:   work.Desired.Image,
+	}
+	klog.V(4).Infof("syncPayload: %s (force=%t)", versionString(desired), work.Desired.Force)
+
+	if work.Attempt == 0 {
+		payload.InitCOUpdateStartTimes()
```

Comment:
I think this is an `apply` thing, and not something that `syncPayload` needs to care about